server/src/models/Review.js

```diff
@@ -11,6 +11,7 @@ class Review extends Model {
       required: ["content"],
       properties: {
         content: { type: "string" },
+        rating: { type: "integer" },
```

Comment:
expand upon validations here if you want to specify that it has to be a positive number or have a min/max value. We want the model json schema validations to fail first and throw an error before postgres throws an unusable error in the application. Whenever we do have database level constraints, we need to make sure they match model validations